### PR TITLE
fix: after each sql sentence is forced to add a semicolon to be deleted

### LINKED ISSUE #978: Remove the code of concating SQL with semicolon in openmldb-batch
Now we will add the semicolon in the end of SQL text which is not required by internal SQL engine now. We should remove the code in `OpenmldbSession.scala`.

```
    var sql: String = sqlText
    if (!sql.trim.endsWith(";")) {
      sql = sql.trim + ";"
    }
```

### PR DESCRIPTION
**Pull request by yabg-shuai666**
close #978

### CODE REVIEW

**Review by jingchen2222 — APPROVED** [review]
LGTM

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala [hunk 1]
```diff
@@ -156,9 +156,7 @@ class OpenmldbSession {
     }
 
     var sql: String = sqlText
```
Comment (tobegit3hub) [inline java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala]: Do not need to use `sql` now. Remove this and use `sqlText` directly.

Comment (yabg-shuai666) [reply java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala]: already edited

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**